graphql-elide-design.md

```diff
@@ -0,0 +1,411 @@
+# GraphQL Support for Elide
+
+## Overview
+
+The next development iteration for Elide focuses on seamless conversion from REST to a graphQL endpoint by simply mounting the provided endpoint to a specified path. Using GraphQL can be a more relevant approach in cases when, say, the user API is not using REST specific services like HATEOAS. Among others, it provides the following key advantages - 
```

Comment:
This implies we are doing away with JSON-API which is not true.  We are going to offer both.